packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -72,6 +72,10 @@ export const messages = {
     'There was an unexpected error when processing the org create response.',
 
   force_org_open_default_scratch_org_text: 'SFDX: Open Default Org',
+  force_org_open_default_scratch_org_container_error:
```

Comment:
It is net new, but the wording mirrors that of the `force_org_create_result_parsing_error` so hopefully it will be an easy approval. I will @mention the doc writer.

Reply:
@hbc-bindu Would you mind taking a look at the text for `force_org_open_default_scratch_org_container_error`? Thanks

Reply:
@hbc-bindu - To provide context, https://github.com/forcedotcom/salesforcedx-vscode/pull/1448#issuecomment-510595001 has the screenshot on where it occurs.

Reply:
Yes, thanks Nick. Just to be clear, the text _"Fail to decipher auth data. reason: Bad input string"_ in that screenshot is taken from the message property of the error returned by the CLI.

However the text in the PR here will be displayed in the same way and place as the whats in the screenshot.

Reply:
@jonnyhork Thanks for providing the context.
You could rephrase the error message as:
An unexpected error occurred while processing the org open response.

Reply:
Sure can, thanks you.